packages/web/src/components/gcds-card/gcds-card.tsx

```diff
@@ -82,51 +97,62 @@ export class GcdsCard {
     this.updateLang();
   }
 
-  private get hasCardFooter() {
-    return !!this.el.querySelector('[slot="footer"]');
+  private get renderDescription() {
+    if (this.el.innerHTML.trim() != "") {
+      return (<slot></slot>);
```

Comment:
It should be fine using an unnamed default slot for formatted text.

Reply:
Do you think it might enhance the self-documenting nature of the component "api" (more clarity for devs and maintainers) as well as provide a level of future-proofing for the component in terms of potential additional slots etc?

Reply:
Yes I could see that it could potentially enhance the self-documenting nature of the component.

With previous components we haven't named the slot if the content passed to it is tied to the main use case of the component, especially if it is the only slot available.